src/firefly/java/edu/caltech/ipac/util/IpacTableUtil.java

```diff
@@ -100,6 +100,7 @@ public static void writeRow(PrintWriter writer, List<DataType> headers, DataObje
             if (v == null && dt.getKeyName().equals(DataGroup.ROWID_NAME)) {
                 v = dt.getFormatInfo().formatData(row.getRowIdx());
             }
+            v = v.replaceAll("\n", " ");
```

Comment:
I am very nervous about make any change to IpacTableUtil unless loi and/or tatiana approve it. Have you talked to them?